developer-docs-site/docs/nodes/measure/node-health-checker.md

```diff
@@ -16,24 +16,22 @@ If you are a node operator, use the NHC service to check if your node is running
 The Aptos Node Health Checker now also supports fullnodes via a central web service with validator support on the way:
 https://nodetools.aptosfoundation.org/#/node_checker
 
-:::tip Node health check for AIT
-If you are participating in the consensus in an Aptos network, you can use the NHC service to verify that your validator node is running correctly. The Aptos team uses this service continuously to check your node's health.
-:::
-
 This document describes how to run NHC locally when you are operating a node.
 
 ## Quickstart
 
 Before you get into the details of how NHC works, you can run the below steps to start the NHC service and send it a request. This quickstart uses a baseline configuration for a devnet fullnode, i.e., it will evaluate your node against a devnet fullnode that is configured with the baseline configuration YAML.
 
-**Important**: If your local node is not a devnet fullnode, e.g. it is a single testnet node for AIT3 registration, you must use a different configuration file. See [the configuration examples in aptos-core](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples) for other such example configs.
+**Important**: If your local node is not a devnet fullnode, you must use a different baseline config. See [the configuration examples in aptos-core](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples) for other such example configs.
```

Comment:
This:
https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/node-checker/configuration_examples
Says, "These instructions are for the Aptos Labs team."





[10:28](https://aptos-org.slack.com/archives/D0464TERLAY/p1677781725306029)
That alright?

Reply:
I think we should consider moving that to internal documentation.